content/derisking-government-tech/resources/kickoff.md

```diff
@@ -1,9 +1,180 @@
 ---
-title: Sample kick-off agenda
+title: Sample kick-off week agenda
 permalink: /derisking-government-tech/resources/kickoff/
 layout: layouts/page
 tags: derisking-government-tech
-sidenav: false
+description: Sample kick-off week agenda for starting work with a vendor development team building custom software for a government agency.
+eleventyNavigation:
+  parent: derisking-government-tech-resources-kickoff
+  key: kickoff
+  order: 1
+  title: Sample kick-off week agenda
+sidenav: true
+sticky_sidenav: true
+subnav:
+- text: Day 1
+  href: '#day-1'
+- text: Day 2
+  href: '#day-2'
+- text: Day 3
+  href: '#day-3'
+- text: Day 4
+  href: '#day-4'
+- text: Day 5
+  href: '#day-5'
```

Comment:
For some reason these nav items aren't bolding when you're on that section:

![image](https://github.com/user-attachments/assets/16139630-447a-4742-8a4c-b71e1d896c2c)

Reply:
I'll look into it. I manually added the anchors for those Day X headings so that I could put them into the table captions, so I may have made an error there.

Reply:
The first issue is that I used `h3`s instead of `h2`s, as you pointed out.

When I fix that, there's another issue: The `setCurrentLink()` JS function that updates the current sub-nav selection works by comparing the current scroll position to the `offsetTop` value for each h2 element on the page, but it assumes that the [`offsetParent`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent) for each h2 is the page's `main` element. However, I put the headings inside each table, which makes the table the h2's offset parent, and breaks the position comparison logic.

The quickest way to address this would be to pull the headings out of the table captions and make them match the nav that we have on other pages. That makes the web experience more consistent with the rest of the site but comes at the expense of visual consistency with the PDF version of the schedule. I'm attaching a screenshot of what that looks like in my local env.

![Screenshot 2024-08-27 162714](https://github.com/user-attachments/assets/0a3d48b6-5e8e-40fe-a4ed-8d068f188150)

If that's not an acceptable trade-off, I can work on updating the setCurrentLink logic to behave differently when the offsetParent is a table. The downside is that it's a hack and it risks introducing bugs into the sub-nav behavior for the existing guides (i.e. it will take longer to manually test).

Reply:
I think the solution in the screenshot is fine, thanks for exploring!